manifests/kube_addons.pp

```diff
@@ -98,7 +96,7 @@
 
   if $schedule_on_controller {
     $schedule_command = ['kubectl', 'taint', 'nodes', $node_name, 'node-role.kubernetes.io/master-']
-    $schedule_onlyif = "kubectl describe nodes ${node_name} | tr -s ' ' | grep 'Taints: node-role.kubernetes.io/master:NoSchedule'"
+    $schedule_onlyif = ['kubectl describe nodes ${node_name} | tr -s " " | grep "Taints: node-role.kubernetes.io/master:NoSchedule"']
```

Comment:
There is an interpolation here that becomes ineffective because of the single quotes. Also i'm not sure there is any additional benefit to using the array for only if given that we are just passing a string.